### ISSUE TITLE
Peer dependancy Gatsby 4.0?

### ISSUE THREAD

**Issue by karldanninger** [benjaminhoffman/gatsby-plugin-segment-js#42]
Will this package work on Gatsby 4?

**Comment by mariusbutuc** [comment]
The latest `v3.7.0` currently displays:

```
warn Plugin gatsby-plugin-segment-js is not compatible with your gatsby version 4.1.0 - It requires gatsby@^3.0.0 || ^2.0.0
```

**Comment by newhouse** [comment]
Yes, this package will work in Gatsby 4. The peer dependency versions were just updated to indicate this and that change was published on npm in version `3.7.1`.
